src/cider/log/appender.clj

```diff
@@ -0,0 +1,133 @@
+(ns cider.log.appender
+  (:require [cider.log.event :as event]))
```

Comment:
Every namespace should have some docstring describing it high-level.

Reply:
@bbatsov I added doc string and metadata information to all non-test namspaces now.

I only added the version metadata (`1.8` I think) to the `cider.nrepl.middleware.log` namespace, since we agreed on the plan to move everything under `cider.log.*` to another repository, so non-NREPL user can benefit from it.